File: 2486/solution.py
class Solution:
    def appendCharacters(self, s: str, t: str) -> int:
        i = 0
        for j in range(len(s)):
            if t[i] == s[j]:
                i += 1
                if i == len(t):
                    return 0
        return len(t) - i

File: 2486/solution.ts
function appendCharacters(s: string, t: string): number {
  let tIndex = 0;
  for (let i = 0; i < s.length; i++) {
    if (s[i] == t[tIndex]) {
      tIndex++;
      if (tIndex == t.length) {
        return 0;
      }
    }
  }
  return t.length - tIndex;
}
